charts/datadog/templates/_ac-agent-sidecar-env.yaml

```diff
@@ -14,12 +14,22 @@
   value: {{ .Values.clusterAgent.admissionController.agentSidecarInjection.provider }}
 {{- end }}
 
+{{- if .Values.clusterAgent.admissionController.containerRegistry }}
+- name: DD_ADMISSION_CONTROLLER_CONTAINER_REGISTRY
+  value: {{ .Values.clusterAgent.admissionController.containerRegistry }}
+{{- else if .Values.registry }}
+- name: DD_ADMISSION_CONTROLLER_CONTAINER_REGISTRY
+  value: {{ .Values.registry }}
+{{- end }}
+
+{{- if .Values.clusterAgent.admissionController.autoInstrumentation.containerRegistry }}
+- name: DD_ADMISSION_CONTROLLER_AUTO_INSTRUMENTATION_CONTAINER_REGISTRY
```

Comment:
If users set DD_ADMISSION_CONTROLLER_AUTO_INSTRUMENTATION_CONTAINER_REGISTRY in `clusterAgent.env` or with --set, which value will win? If we are overwriting values that some customers had set manually with  a different value in `.Values.clusterAgent.admissionController.autoInstrumentation.containerRegistry`, we might break their instrumentations.